code/modules/mob/living/simple_animal/aliens/statue.dm

```diff
@@ -0,0 +1,428 @@
+		for(var/obj/structure/mirror/M in range(3, check)) //Weeping angels hate mirrors. Probably because they're ugly af
+			if ((!M.shattered )||(!M.glass))
+				annoyance += 3
+				if (prob(5) && (ai_inactive == 0))
+					AI_mirrorshmash()
+					annoyance -= 50
+				return src //if it sees the mirror, it sees itself, right?
+	return null
+
+// Cannot talk
+
+/mob/living/simple_animal/hostile/statue/say()
+	return 0
+
+// Turn to dust when gibbed
+
+/mob/living/simple_animal/hostile/statue/gib()
+	dust()
+
+
+// Stop attacking clientless mobs
+
+/mob/living/simple_animal/hostile/statue/proc/CanAttack(atom/the_target) //ignore clientless mobs
+	if(isliving(the_target))
+		var/mob/living/L = the_target
+		if(!L.client && !L.ckey)
+			return 0
+	return ..()
+
+// Statue powers
+
+// Flicker lights
+/spell/aoe_turf/flicker_lights
+	name = "Flicker Lights"
+	desc = "You will trigger a large amount of lights around you to flicker."
+	spell_flags = 0
+	charge_max = 400
+	range = 10
+
+/spell/aoe_turf/flicker_lights/cast(list/targets, mob/user = usr)
+	for(var/turf/T in targets)
+		for(var/obj/machinery/light/L in T)
+			L.flicker()
+	return
+
+//Blind AOE
+/spell/aoe_turf/blindness
+	name = "Blindness"
+	desc = "Your prey will be momentarily blind for you to advance on them."
+
+	message = "<span class='notice'>You glare your eyes.</span>"
+	charge_max = 250
+	spell_flags = 0
+	range = 10
+
+/spell/aoe_turf/blindness/cast(list/targets, mob/user = usr)
+	for(var/mob/living/L in living_mob_list)
+		if(L == user)
+			continue
+		var/turf/T = get_turf(L.loc)
+		if(T && T in targets)
+			L.Blind(4)
+	return
+
+
+/spell/aoe_turf/shatter
+	name = "Shatter mirrors!"
+	desc = "That handsome devil has to wait. You have people to make into corpses."
+
+	message = "<span class='notice'>You glare your eyes.</span>"
+	charge_max = 2000
+	silenced = 500
+	spell_flags = 0
+	range = 10
+
+
+
+
+/spell/aoe_turf/shatter/cast(list/targets, mob/user = usr)
+	for(var/obj/structure/mirror/M in range(5, src))
+		if ((!M.shattered )||(!M.glass))
+			M.shatter()
+	return
+
+
+
+/mob/living/simple_animal/hostile/statue/verb/toggle_darkness()
+	set name = "Toggle Darkness"
+	set desc = "You ARE the darkness."
+	set category = "Abilities"
+	seedarkness = !seedarkness
+	plane_holder.set_vis(VIS_FULLBRIGHT, !seedarkness)
+	to_chat(src,"You [seedarkness ? "now" : "no longer"] see darkness.")
+
+
+
+/mob/living/simple_animal/hostile/statue/restrained()
+	. = ..()
+	if(can_be_seen(loc))
+		return 1
+
+
+
+/mob/living/simple_animal/hostile/statue/ListTargets()
+	. = ..()
+	for(var/mob/living/V in mobs_in_xray_view(20, src))
+		if(V == creator)
+			continue
+		. += V
+	return .
+
+
+
+/obj/item/cursed_marble
+	name = "marble slab"
+	desc = "A peculiar slab of marble, radiating with dark energy."
+	icon = 'icons/obj/stacks.dmi'
+	icon_state = "sheet-marble"
+	description_info = "Summons the Statue - mysterious powerful creature, that can move only when unsurveyed by living eyes."
```

Comment:
should it have 'a' before mysterious?

Reply:
a is needed, yes, but the comma is not necessary.